libs/blocks/section-metadata/section-metadata.css

```diff
@@ -183,4 +183,19 @@ main>.section[class*='-up']>.content {
     grid-template-columns: repeat(5, 1fr);
   }
 
+  .section.grid-template-columns-1-2 {
+    grid-template-columns: 1fr 2fr;
+  }
+  
+  .section.grid-template-columns-2-1 {
+    grid-template-columns: 2fr 1fr;
+  }
+
+  .section.grid-template-columns-1-3 {
+    grid-template-columns: 1fr 3fr;
+  }
+  
+  .section.grid-template-columns-3-1 {
+    grid-template-columns: 3fr 1fr;
+  }
```

Comment:
I love the use of `fr`unit. My only concern here is related to the `grid-template-columns` and all of the possible variations needed to support a full 12 col grid system in the future. Would Flexbox be more straightforward? Thoughts?

Reply:
I believe with Flexbox we would have to have child items of the parent declare their widths. So we'd have to add classes to children in the code and additional classes for widths like `span-4-col` right?.

With the `grid-template-columns` the parent container controls the children, and we already have the makings of a system with classes like `two-up` and `three-up`. But to your point, we may haver to add quite a few classes to account for variants. Perhaps if we get to that point we could actually construct the value for `grid-template-columns` with JS?

Reply:
>I believe with Flexbox we would have to have child items of the parent declare their widths. So we'd have to add classes to children in the code and additional classes for widths like span-4-col right?.

Yeah, i think there is some give and take here. I just wanted to throw the thought out there for people to think about.